docker/sandbox/flyte-entrypoint-default.sh

```diff
@@ -44,4 +44,7 @@ helm upgrade -n flyte --create-namespace flyte $charts --kubeconfig /etc/rancher
 
 wait-for-flyte.sh
 
+# With flytectl sandbox --source flag, we mount the root volume to user source dir that will create helm & k8s specific directory. It cause issue while doing fast register 
```

Comment:
Can we explain briefly what issue it causes

Reply:
There is no issue with mac but on linux these files mount as root user and pyflyte through error during fast register(`permission denied while reading these files`)

Reply:
Lets mention this issue so its clear from the code aswell that its only on linux that we have a particular issue.
Also if there is a link to the github issue for linux since this works on mac correctly and a long term fix for it